### PR TITLE
Remove scripted Opex Template

### PR DESCRIPTION
**Pull request by ImplicitDeny**
Remove all files linked to the script version of Opex template due to the upcoming mod version GSRI Opex https://github.com/team-gsri/GSRI-Opex

### CODE REVIEW

File: FORM_Tembelan.Tembelan/init.sqf [hunk 1]
```diff
@@ -1,5 +1 @@
-// Reminder : add D-651 Normandie tag french flag with Eden properties
-
-// Must be executed on all clients
-[destroyer] call GSRI_fnc_libertyInit;
 nopop = true;
```
Comment (ArwynFr) [inline FORM_Tembelan.Tembelan/init.sqf:1]: is it useful ?

Comment (ImplicitDeny) [reply FORM_Tembelan.Tembelan/init.sqf:1]: What do you mean, this "nopop" stuff ? I didn't pay attention to this, as it is not linked to my opex template. It is usually used as a variable name to control practice targets behavior, so it's probably used by @legcs in other scripts.

Comment (ArwynFr) [reply FORM_Tembelan.Tembelan/init.sqf:1]: https://github.com/team-gsri/Tembelan/search?q=nopop yields no other results :(

Comment (ImplicitDeny) [reply FORM_Tembelan.Tembelan/init.sqf:1]: What about init fields in mission.sqm ?

Comment (ArwynFr) [reply FORM_Tembelan.Tembelan/init.sqf:1]: `nopop` cannot be found in mission.sqm

Comment (ImplicitDeny) [reply FORM_Tembelan.Tembelan/init.sqf:1]: Found ! Nopop is an old global variable used to influence the behavior of all practice targets, preventing them to be raised by default and to rise back when hit. However this very dirty work was modified as early as 2014, and is now replaced by an individual control based on AnimationSources. Consequently it is not needed anymore, removing it.

**Pull request merged**